hailjwt/test/test_jwt.py

```diff
@@ -52,3 +52,9 @@ def test_modified_cyphertext_is_error():
 
 def test_get_domain():
     assert hj.get_domain('notebook.hail.is') == 'hail.is'
+
+
+def test_unsafe_decode():
+    c = hj.JWTClient(hj.JWTClient.generate_key())
```

Comment:
You should either support token=None, or make unsafe_decode be `@staticmethod` since it doesn't require state.  The generate_key shouldn't be here.

Reply:
It's needed to encode the json -- right? I made it a static method.